src/main/resources/assets/thaumicaugmentation/lang/en_us.lang

```diff
@@ -500,20 +500,20 @@ thaumicaugmentation.research.ward_openers.1=I have successfully created warded v
+thaumicaugmentation.research.impetus.0=With the wisdom I have gained from the scrolls of old, I believe I am now ready to fully unlock the potential of impetus. Previously I was able to use it to create void seeds, and after that collect and storage through manual use of a caster's gauntlet, but with only a little more research I should be able to use the turbulent forces of the Void to do a great deal more than that.<BR>I already have a recipe for what is known as an Impetus Jewel, and while I know what it does in theory, I must now actually create one to see how it behaves in practice. The jewel, however, will only be capable of resonating impetus to receive and transmit it directly; therefore, I still need a way to draw impetus out of a flux rift in the first place. Fortunately, I have already done this previously, firstly with a Void Siphon and secondly with an Impetus Conductor. The former seems more appropriate a starting point for what I am aiming for, that being automatic collection of raw impetus. However, a Void Siphon requires a nether star, so I should probably endeavor to find a way to avoid needing them at all.<BR>Despite the effort this will take, these next steps will still be far simpler than getting past the dead-end I faced previously. At least I now have all of the fundamental knowledge required to actually do this.
```

Comment:
I agree that the original wording was awkward with "... and after that collection and storage through manual use of a caster’s gauntlet", but "collect and storage" doesn't sound great either. Maybe "collect and store it"?